collect_app/src/main/java/org/odk/collect/android/activities/InstanceUploaderActivity.java

```diff
@@ -80,7 +80,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
         setTitle(getString(R.string.send_data));
 
-        // get any simple saved state...
+        // load any simple saved state...
```

Comment:
I think this may have been the result of over-zealous search and replace. I see it in some other files as well.

Reply:
Well that makes me sad. I went through and killed each one. I'll wait until everything else is finished to clean things up again.